api/pom.xml

```diff
@@ -77,13 +74,17 @@
                 <groupId>org.jsonschema2pojo</groupId>
                 <artifactId>jsonschema2pojo-maven-plugin</artifactId>
                 <configuration>
-                    <sourceDirectory>${basedir}/src/main/resources/schema</sourceDirectory>
+                   <sourceDirectory>${basedir}/src/main/resources/schema</sourceDirectory>
+                   <!-- <sourcePaths>
+                      <sourcePath>https://raw.githubusercontent.com/serverlessworkflow/specification/main/schema/workflow.yaml</sourcePath>
+                    </sourcePaths> -->
```

Comment:
good catch @ricardozanini
I leave that on purpose because that should work, but its not because of the pojo generator usage of apache commons. If we fix that issue, we can uncomment that code and use directly the real schema rather than copying it

Reply:
Just a comment on top of this comment then 👍